src/token/src/canister/inspect.rs

```diff
@@ -131,8 +112,10 @@ fn inspect_message() {
             // We reject this message, because a call with cycles cannot be made through ingress,
             // only from the wallet canister.
         }
-        _ => {
-            ic_cdk::println!("The method called is not listed in the access checks. This is probably a code error.");
+
+        "getSignedTransaction" => ic_cdk::api::call::accept_message(),
```

Comment:
We reject it alway, for the call must be done though a wallet, so it should never be inpected.